lib/ctblgrp.gi

```diff
@@ -1756,9 +1738,9 @@ StandardClassMatrixColumn := function(D,M,r,t)
       p:=RepresentativeAction(Stabilizer(orb.group,r),c,t);
       if p<>fail then
 	# was the first column of the galois class active?
-	if ForAny(M,i->i[c]>0) then
+	if ForAny([1..NrRows(M)],i->M[i,c]>2) then
```

Comment:
Just for curiosity: The check for `>0` gets replaced by a check for `>2`; correct?

Reply:
Thank you for spotting. No, this is an editing error (even though its effects are harmless and only introduce tiny inefficiencies). Wil fix.